physionet-django/user/models.py

```diff
@@ -464,6 +464,28 @@ def get_users_with_permission(app_label, permission_codename):
 
         return users
 
+    def get_credentialing_status(self):
+        """
+        Returns the credentialing status of the user.
+        If the user is not credentialed, returns the status of the user's credentialing application.
+        """
+        if self.is_credentialed:
+            return 'Credentialed'
+        else:
+            application = self.credential_applications.last()
+            if not application:
+                return 'No application found'
+            application_status = application.status
+            # pending applications should return the review status
+            if application_status == 0:
```

Comment:
I don't think that sharing the statuses below is appropriate (rejected. withdrawn, revoked). In these cases, it is enough to say that the user is not credentialed ("No application found" would be fine).

Reply:
Noted. Let me do that.